bedrock/firefox/views.py

```diff
@@ -612,6 +613,8 @@ def get_template_names(self):
                 template = 'firefox/developer/whatsnew.html'
             else:
                 template = 'firefox/whatsnew/index.html'
+        elif version.startswith('83.') and ftl_file_is_active('firefox/whatsnew/whatsnew-fx83'):
+            template = 'firefox/whatsnew/whatsnew-fx83.html'
```

Comment:
Should this have the same sort of `locale.startswith('en-'):` logic for the supported languages from WNP82?

Reply:
IMHO the `ftl_file_is_active` is enough of a check.

Reply:
Is there a designated someone who will let Peying know which locales need to be activated?

Reply:
`ftl_file_is_active` is always `True` for English locales.